be/src/formats/parquet/group_reader.h

```diff
@@ -40,21 +40,22 @@ namespace starrocks::parquet {
 struct GroupReaderParam {
     struct Column {
         // parquet field index in root node's children
-        int32_t field_idx_in_parquet;
-
-        // column index in chunk
-        int32_t col_idx_in_chunk;
+        int32_t idx_in_parquet;
 
         // column type in parquet file
-        tparquet::Type::type col_type_in_parquet;
+        tparquet::Type::type type_in_parquet;
 
-        // column type in chunk
-        TypeDescriptor col_type_in_chunk;
+        // column index in chunk
+        int32_t idx_in_chunk;
```

Comment:
how about giving all fields a default value of -1?

Reply:
OK.

but this field  will be removed in next pr. in this commit https://github.com/StarRocks/starrocks/pull/42087/commits/d54f396f7a2a8ace966077141b71e614f82ff952